components/dashboard/src/projects/Projects.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Link } from "react-router-dom";
+import Header from "../components/Header";
+import projectsEmpty from '../images/projects-empty.svg';
+
+export default function() {
+    return <>
+        <Header title="Projects" subtitle="Manage recently added projects." />
+        <div>
+            <img alt="Projects (empty)" className="h-44 mt-24 mx-auto" role="presentation" src={projectsEmpty} />
```

Comment:
**praise**: This looks so good, that I added https://github.com/gitpod-io/gitpod/issues/4455. 😋